docs/docs/development.md

```diff
@@ -63,7 +63,7 @@ mkdir -p data
 Initialize the `qhub-config.yaml`
 
 ```shell
-python -m qhub init local --project=thisisatest  --domain qhub.test --auth-provider=password --terraform-state=local
+python -m qhub init local --project=thisisatest  --domain github-actions.qhub.dev --auth-provider=password --terraform-state=local
 ```
 
 Give a password to the default user. For this the example password is
```

Comment:
Where do I import this? Is using `bcrypt` mandatory or can I choose a personal password?

Reply:
You can install `bcrypt` with pip (https://pypi.org/project/bcrypt/). I think you should be able to choose a personal password.

Reply:
Alright, I got that bcrypt is just a python package that will help composing the password.
My question is **where to import it?**. The user is asked to import the package but not told where to do it. It is not clear for me if I should use `bcrypt` on my own and copy and paste the generated password o n the CLI command or if I have to import the package somewhere in QHub to have that working.

Sorry the highlighting is not helping more.